### ISSUE TITLE
MenuBar - Accessibility issue on menu items

### ISSUE THREAD

**Issue by antsteyer** [primefaces/primevue#7704]
### Describe the bug

MenuBar component menu items are not accessible due to the attribute `aria-level`. This attribute is not valid for the role `menuitem`, leading to a critical accessibility issue https://dequeuniversity.com/rules/axe/4.10/aria-allowed-attr.

![Image](https://github.com/user-attachments/assets/1569c7cb-1888-49d4-aaee-ff90a97639cc)

Accessibility is very important for our business, please consider merging the PR quick if possible, thanks !

### Pull Request Link

https://github.com/primefaces/primevue/pull/7703

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primevue.org/menubar/

### Environment

Chrome navigator and lighthouse tool

### Vue version

3.5.13

### PrimeVue version

4.3.4

### Steps to reproduce the behavior

1. Go to https://primevue.org/menubar/
2. Open developer console
3. Run Lighthouse - Accessibility

### Expected behavior

MenuBar component should be accessible
